### ISSUE TITLE
@fromQuery@ doesn't work with PostgreSQL

### ISSUE THREAD

**Issue by Zawuza** [viant/dsunit#12]
Hello!

First of all, thanks for a cool test framework, I really like it and appreciate your work!

I discovered a small problem while using the framework. Here is a sample "expect"-JSON that can be used with a database:
```
[
    {
        "@fromQuery@": "SELECT bla_bla FROM table"
    },
    {
        "bla_bla": 2
    }
]
```
The issue is that if used with PostgreSQL, it raises the following error: `failed to prepare sql: SELECT bla_bla FROM (SELECT bla_bla FROM table) with [] due to:pq: subquery in FROM must have an alias`. I discovered that dsunit constructs a query without an alias, which is required(?) in PostgreSQL. I found a workaround, it's possible to modify the JSON file like this:
What is used now:
```
[
    {
        "@fromQuery@": "SELECT bla_bla FROM table) AS new_table;--"
    },
    {
        "bla_bla": 2
    }
]
```
In this case, everything will work as expected. So, it looks like in case of PostgreSQL dsunit should add `AS new_table` for query to work. My PostgreSQL version is 11.2.

**Comment by adranwit** [comment]
Patched in viant/dsc v0.10.2 (https://github.com/viant/dsc/releases)
https://github.com/viant/dsc/blob/master/table_test.go

**Comment by Zawuza** [comment]
I confirm that it works now for my tests! 
Thank you very much! :)
